### PR TITLE
confs-proofread

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
Proofread conf files comments.

### CODE REVIEW

File: etc/nebula-graphd.conf.default [hunk 1]
```diff
@@ -1,24 +1,24 @@
 ########## basics ##########
-# Whether to run as a daemon process
+# Enabling/Disabling daemon process
```
Comment (dangleptr) [inline etc/nebula-graphd.conf.default:2]: To be honest, i prefer the original expression.

Comment (Amber1990Zhang) [reply etc/nebula-graphd.conf.default:2]: but i think the modified are more clear.

File: etc/nebula-graphd.conf.default [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
 
 ########## logging ##########
-# The directory to host logging files, which must already exists
+# Specify the path where log is written. The path must exist
 --log_dir=logs
-# Log level, 0, 1, 2, 3 for INFO, WARNING, ERROR, FATAL respectively
+# Specify the log severity levels. 0, 1, 2, 3 for INFO, WARNING, ERROR, FATAL respectively
 --minloglevel=0
-# Verbose log level, 1, 2, 3, 4, the higher of the level, the more verbose of the logging
+# Specify the verbose log severity levels. Available levels are 1, 2, 3, 4. The higher the level, the more verbose of the log
```
Comment (critical27) [inline etc/nebula-graphd.conf.default:12]: 01234, or explain about 0

File: etc/nebula-graphd.conf.production [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 --pid_file=pids/nebula-graphd.pid
 
 ########## logging ##########
-# The directory to host logging files, which must already exists
+# Specify the path where log is written. The path must exist
 --log_dir=logs
 # Log level, 0, 1, 2, 3 for INFO, WARNING, ERROR, FATAL respectively
 --minloglevel=0
-# Verbose log level, 1, 2, 3, 4, the higher of the level, the more verbose of the logging
+# Specify the verbose log severity levels. Available levels are 1, 2, 3, 4. The higher the level, the more verbose of the log
```
Comment (critical27) [inline etc/nebula-graphd.conf.production:12]: ditto

File: etc/nebula-metad.conf.default [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
 
 ########## logging ##########
-# The directory to host logging files, which must already exists
+# Specify the path where log is written. The path must exist
 --log_dir=logs
-# Log level, 0, 1, 2, 3 for INFO, WARNING, ERROR, FATAL respectively
+# Specify the log severity levels. 0, 1, 2, 3 for INFO, WARNING, ERROR, FATAL respectively
 --minloglevel=0
-# Verbose log level, 1, 2, 3, 4, the higher of the level, the more verbose of the logging
+# Specify the verbose log severity levels. Available levels are 1, 2, 3, 4. The higher the level, the more verbose of the log
```
Comment (critical27) [inline etc/nebula-metad.conf.default:12]: ditto

**Pull request closed without merging**